dev-docs/bidders/screencore.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: bidder
+title: Screencore
+description: Prebid Screencore Bidder Adaptor
+biddercode: screencore
+gdpr_supported: true
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+media_types: banner, video, native
+safeframes_ok: true
+deals_supported: true
+pbs: true
+sidebarType: 1
+floors_supported: true
+prebid_member: false
+fpd_supported: false
+gvl_id: none
+multiformat_supported: will-bid-on-one
+ortb_blocking_supported: true
+userIds: all
+---
+
+### Note:
+
+The Example Bidding adapter requires setup before beginning. Please contact us at connect@screencore.io.
+Screencore will bid on first impression in request.
```

Comment:
Please elaborate on this. It seems quite limiting that you only bid on the first request. Why not just have your adapter split up multi-imp requests?

Reply:
yes, we will process only first impression in request. We know that it's quite limiting but we're planning to add multi-imp processing in the future. We also work with with all supply platforms in a similar way, using only first impression in request